paddle/gserver/layers/ContextProjection.cpp

```diff
@@ -13,8 +13,8 @@ See the License for the specific language governing permissions and
 limitations under the License. */
 
 #include "ContextProjection.h"
```

Comment:
Either in this PR or in the next one, we need to use full path with `#includes`.  For example,

```cpp
#include "ContextProjection.h" 
```

should we 

```cpp
#include "paddle/gserver/layers/ContextProjection.h"
```

This change should be done together with the change of CMakeLists.txt files to remove some pre-defined C++ include paths.